scilog/src/app/core/remote-data.service.ts

```diff
@@ -119,7 +119,11 @@ export class LogbookItemDataService extends RemoteDataService {
       config.filter.tags.forEach(tag => {
         whereFilter.push({ "tags": tag })
       })
-
+    }
+    if ((config.filter.excludeTags) && (config.filter.excludeTags.length > 0)) {
```

Comment:
would the safety operator work here? 
`config.filter.excludeTags?.length > 0`

Reply:
thanks, I changed it!